crowbar_framework/app/controllers/application_controller.rb

```diff
@@ -215,4 +216,17 @@ def upgrade
       end
     end
   end
+
+  def check_eol
+    branding = YAML.load_file(Rails.root.join("config", "branding.yml"))
+    return if branding["eol_date"].blank?
+    return if Time.parse(branding["eol_date"]).future?
+
+    flash[:alert] = I18n.t(
+      "dashboard.eol",
+      product: branding["page_title"],
```

Comment:
In the case of this PR yes, but the branding (page_title) will be replaced in our package. But I'm unsure if we should set the EOL date in here or in our package.